mitodl/mitxpro
# Fixed broken JS-based interactive elements on product page

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2082

#### What's this PR do?
Fixes bug where JS based interactive elements were broken in the product detail page

#### How should this be manually tested?
Load a course page with multiple future course runs. You should see a "More Dates" link. Clicking that link should bring up a tooltip with the future dates

#### Any background context you want to provide?
The js file I deleted was not being used anymore

#### Screenshots (if appropriate)
![ss 2021-01-22 at 15 44 20 ](https://user-images.githubusercontent.com/14932219/105544320-d7488200-5cc8-11eb-8f18-be997e802b4a.png)


#### What GIF best describes this PR or how it makes you feel?
(Optional)
